README.md

```diff
@@ -97,6 +83,17 @@ If you'd like to chat, we have a [community slack](http://amserializers.herokuap
 
 Thanks!
 
+## Documentation
+
+- [0.10 (master) Documentation](https://github.com/rails-api/active_model_serializers/tree/master)
+  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/v0.10.0)
```

Comment:
though @remear it appears you did not `rake release` because there is no `v0.10.0` tag